### PR TITLE
Add VBA cue to code block.

### PR DESCRIPTION
**Pull request by hecon5**
Add Code cue to code block and add subroutine definition.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@hecon5 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@hecon5 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/ado/guide/data/command-object-parameters.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ ms.author: jroth
 # Command Object Parameters
 The previous topic discussed [Creating and Executing a Simple Command](./creating-and-executing-a-simple-command.md). A more interesting use for the [Command](../../reference/ado-api/command-object-ado.md) object is shown in the next example, in which the SQL command has been parameterized. This modification makes it possible to reuse the command, passing in a different value for the parameter each time. Because the [Prepared Property](../../reference/ado-api/prepared-property-ado.md) property on the **Command** object is set to **true**, ADO will require the provider to compile the command specified in [CommandText](../../reference/ado-api/commandtext-property-ado.md) before executing it for the first time. It also will retain the compiled command in memory. This slows the execution of the command slightly the first time it is executed because of the overhead required to prepare it, but results in a performance gain each time the command is called thereafter. Therefore, commands should be prepared only if they will be used more than one time.  
   
-```  
+```VBA
```
Comment (rwestMSFT) [inline docs/ado/guide/data/command-object-parameters.md:20]: @hecon5 Please make this lowercase, and then I'll approve your change.

**Comment by rwestMSFT** [comment]
#reassign:rwestMSFT

File: docs/ado/guide/data/command-object-parameters.md (hunk 1 above)
Comment (hecon5) [reply docs/ado/guide/data/command-object-parameters.md]: Done!

**Comment by PRMerger7** [comment]
@hecon5 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rwestMSFT** [comment]
#sign-off

**Comment by PRMerger5** [comment]
@rwestMSFT: I'm sorry - only the author of this article, @rothja, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by rothja** [comment]
#sign-off

**Comment by rwestMSFT** [comment]
@hecon5 Thanks again for the contribution!

**Pull request merged**